README.md

```diff
@@ -50,9 +58,8 @@ For more information, please refer to:
 └── template.json
 ```
 
-We use `Typescript` by default for our React applications. Along with the standard files from a `create-react-app` 
-project, the folder structure in the `src` folder is created as per the
-[React Conevention](https://nimblehq.co/compass/development/code-conventions/react/#project-structure).
+`Typescript` is used by default for our React applications. To the standard files from a non-ejected `create-react-app` project, this templates adds a folder structure in `/src` that follows our
+[React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).
```

Comment:
I think the line break is a little bit odd, maybe you can consider:

Having a soft line break between the first 2 sentences: (also fix the `template` in singular form)
```suggestion
`Typescript` is used by default for our React applications.  
To the standard files from a non-ejected `create-react-app` project, this template adds a folder structure in `/src` that follows our [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).
```

or just inlining:

```suggestion
`Typescript` is used by default for our React applications. To the standard files from a non-ejected `create-react-app` project, this template adds a folder structure in `/src` that follows our [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).
```

Reply:
✅ 0ac66df

I've found another bad line break (I think it comes from the auto-linter of some IDE... like auto line break after XX chars).

Should be all fixed now ;-) Thanks!